index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Hello, Welome!</title>
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body>
+    <header> 
+      <nav>
+         <p>Welcome!  <img src="/home/edwin/Portfolio-webpage/Images/Icon - Menu.png" class="image">  </p>
```

Comment:
- [ ]  Please implement the suggestions the previous code reviewers gave. Kindly fix the path to the images. The image is broken and not shown due to an incorrect path; it should be ./Images/Icon-Menu.png instead of /home/edwin..../... same for all the images :+1: